docs/agent/cli.mdx

```diff
@@ -331,6 +331,7 @@ The `ngrok http` command is used to start a tunnel listening for HTTP/HTTPS traf
 | Flag                               | Description                                                                                       |
 | ---------------------------------- | ------------------------------------------------------------------------------------------------- |
 | `--authtoken string`               | ngrok authtoken                                                                                   |
+| `--app-protocol string`            | specify the application protocol to be used: `http1`, `http2` (default `http1`)                   |
```

Comment:
this description is better than the one from the CLI: `--app-protocol string              Set application protocol: 'http2'` follow on would be to update the cli text on the next release: fyi @salilsub